testapps/test_pillowtop/tests/test_app_pillow.py

```diff
@@ -102,7 +102,7 @@ def test_prune_autogenerated_builds(self):
 
         # Build #2, auto-generated
         app.save()
-        build2 = make_async_build_v2(app.get_id, app.domain, app.version)
+        build2 = make_build(app, app.domain, app.version, 'someone', async=False)
```

Comment:
looks like this method never returns anything useful, so this test probably fails now?

Reply:
Also these parameters are out of order

Reply:
@proteusvacuum https://github.com/dimagi/commcare-hq/pull/24860/commits/45e45e8e711dadff8dbef321587773c10c098f5b

Reply:
@esoergel https://github.com/dimagi/commcare-hq/pull/24860/commits/ffdef9680a1e40698b45cbc793462dad24159f64